manifests/init.pp

```diff
@@ -42,6 +43,7 @@
     creates => "${download_dir}/go-${version}.tar.gz",
     unless  => "which go && go version | grep '${version}'",
     require => Package['curl'],
+    timeout => ${download_timeout},
```

Comment:
I may be wrong but isn't the standard style to only wrap variables like this when they are added into strings? In which case this would simply be `$download_timeout`